napari/_qt/menus/_tests/test_file_menu.py

```diff
@@ -1,11 +1,16 @@
 from unittest import mock
 
+import numpy as np
```

Comment:
Now that the `file_menu` no longer exists in this folder, I wonder if we should move the tests... I just imagine eventually this folder might be empty except for tests? Maybe we should make a subfolder under `_qapp_model` 😬 idk

Reply:
Well `napari/_qt/_qapp_model/_tests/` already exists (with `test_qapp_model_menus.py`). We could add menu tests there? Seems like we are testing only qactions (since menu app only actions are defined elsewhere).

Reply:
Have moved the file as described above.

Reply:
Note this mucks up the git diff. Maybe I could keep the file where it is in this PR (to aid review) and move it in another PR? WDYT @DragaDoncila

Reply:
@lucyleeow yep, fine by me!